content/en/guides/employee-engagement.md

```diff
@@ -2,198 +2,201 @@
+Open Source projects are made by people, with people, for people. It's easy to forget that the online avatars are real people. You should always remain respectful and follow the defined rules of engagement by the community.
 
-Projects and communities often have a Code of Conduct, it is a pledge from the community leadership towards the community that no abusive behavior will be tolerated. Take a moment to read and understand them. You need to follow this the same way you need to follow your internal employee handbook.
+Projects and communities often have a Code of Conduct. It is a pledge from the community leadership towards the community that no abusive behavior will be tolerated. Take a moment to read and understand them. You need to follow this the same way you need to follow your internal employee handbook.
 
-Failing to follow such guidelines always has consequences for both your reputation but also the your company's. It's on everyone's interest (your and your company) that you always engage in a respectful manner. Your company will take any infringement of open source projects' code of conducts as if they were infringements of the internal employee handbook.
+If you do not follow the guidelines there will be consequences for both your reputation and your company's. It's in everyone's interest that you always engage in a respectful manner. Your company will take any infringement of open source projects' code of conducts as if they were infringements of the internal employee handbook.
 
-But it is not only about code of conducts, each community has their preferred way of communication and way of engage in discussions, please make sure to understand how the communities you are part of prefer to be engaged and follow their ways.
+In addition to codes of conduct each community has its preferred way of communication and ways of engaging in discussion. Take time to understand how each community prefers to be engaged and follow their ways.
 
-Also be aware that open source communities often have people from very diverse backgrounds, coming from different cultures, speaking different languages. This can be hard to notice, understand and handle, especially when using communication channels such as email or bug reports, which don't convey much context. It can be helpful to think of that in the light of Postel's law: "Be conservative in what you do, be liberal in what you accept from others."
+Be aware that open source communities often have people from very diverse backgrounds who come from different cultures and speak different languages. This can be hard to notice when using written communication channels which don't contain much context. Postel's law is a useful guide: "Be conservative in what you do, be liberal in what you accept from others". 
 
-### Suggested tactics for enforcing respectful behavior
+### Tips for demonstrating respectful behavior
```

Comment:
To be consistent with the name of the principle we might want to say promoting respectful behavior here as well.

Reply:
Good catch. I prefer it to be consistent as you suggest.